packages/status-js/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@status-im/core",
+  "name": "@status-im/js",
```

Comment:
Regular full-text search didn't pick this up. Another reason why I think `status-` would help.